x-pack/plugins/alerts/server/plugin.ts

```diff
@@ -295,6 +325,18 @@ export class AlertingPlugin {
     };
   };
 
+  private createAlertClientHandlerContext = (core: CoreSetup) => {
+    const { alertsClientFactory } = this;
+    return async function alertsFakeContext(request: KibanaRequest) {
+      const [{ savedObjects }] = await core.getStartServices();
+      return {
+        getAlertsClient: () => {
+          return alertsClientFactory!.create(request, savedObjects);
+        },
+      };
+    };
+  };
+
```

Comment:
I might be missing something... why do we need this if we have `createRouteHandlerContext`?
Seems like unnecessary duplication 🤔